test/Groups/matrixgroups.jl

```diff
@@ -139,6 +139,9 @@ end
      @test GAP.Globals.IsBijective(f)
      @test order(G) == GAP.Globals.Order(H)
    end
+
+   G = MatrixGroup(2, QQ, dense_matrix_type(QQ)[])
+   @test order(isomorphic_group_over_finite_field(G)[1]) == 1
```

Comment:
```suggestion
   @test order(Oscar.isomorphic_group_over_finite_field(G)[1]) == 1
```

Reply:
I guess you cannot add it to the `inputs` above?

Reply:
Thanks! And no, you can't do `matrix_group([])`, so I can't add the empty array to the inputs of the for loop.